### ISSUE TITLE
DeviantArt: Downloading a DA artist directory only downloads Featured images, not all images

### ISSUE THREAD

**Issue by Phenrei** [4pr0n/ripme#63]
If you point directly to the full image collection of an artist instead of a subgallery, it only downloads the featured section (sometimes empty, usually not every image) instead of the all images category because the sanitizer filters out the "?catpath=/" part.

I coded a half-fix (it breaks the rip stopping on its own for a reason I can't figure out) that is here: https://github.com/Phenrei/ripme/commit/8a68c79047173a911b4bab523407cec6d0cce5af (ignore the first change).

Short version of it is a quick regex to see if it's base directory or not, and if so keep the ?catpath=/ in all rip URLs. Feel free to clean up, fix my bug, and take all credit.

**Comment by rautamiekka** [comment]
https://github.com/4pr0n/ripme/issues/550

**Comment by metaprime** [comment]
@rautamiekka Is this the same issue or just similar? Can we leverage the change posted here?

**Comment by rautamiekka** [comment]
^ Similar.
